app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -188,6 +188,25 @@ export const buildAggregatedDateLastModifiedDate = (
   };
 };
 
+/**
+ * Build props for data use restriction BasicCell component from the given files response.
+ * @param filesResponse - Response model return from files API.
+ * @returns model to be used as props for the BasicCell component
+ */
+export const buildAggregatedDataUseRestriction = (
+  filesResponse: FilesResponse
+): React.ComponentProps<typeof C.BasicCell> => {
```

Comment:
Sorry @jpaten - I might have given you incorrect suggestion for the `buildAggregatedDataUseRestriction`. I wonder if we should make this an `NTagCell` and return all possible values from the `filesResponse`?

If so, update:
- lines 192 and 194 with `NTagCell` in the comments.
- line 198 with `C.NTagCell`
- the return values to be an array of values.